src/clients/bridges/BaseAdapter.ts

```diff
@@ -310,6 +312,7 @@ export abstract class BaseAdapter {
       amount,
       contract: contract.address,
       txnRequestData,
+      mrkdwn: `Sent ${formatUnitsForToken(tokenSymbol, amount)} ${tokenSymbol}`,
```

Comment:
Worth including the chainId in the log message?

```suggestion
      mrkdwn: `Sent ${formatUnitsForToken(tokenSymbol, amount)} ${tokenSymbol} to chain ${this.chainId}`,
```